### PR TITLE
modification fichier index.js index.html main.py

### PR DESCRIPTION
**Pull request by cosnelle**

### CODE REVIEW

File: Graphe_chartjs/index.js [hunk 1]
```diff
@@ -1,56 +1,71 @@
...
       mode_transport_univ.addEventListener('click', change_mode_transport_univ)
    
+      const pourcent_mode_transport_univ  = document.getElementById('pourcent_mode_transport_univ');
+      pourcent_mode_transport_univ .addEventListener('click', change_pourcent_mode_transport_univ )
+
       const nb_person_univ = document.getElementById('nb_person_univ');
       nb_person_univ.addEventListener('click', change_nb_person_univ)
 
+
+      const autocolors = window['chartjs-plugin-autocolors']; // génération de couleurs automatique pour les différents graphiques
```
Comment (damienmarchal) [inline Graphe_chartjs/index.js:17]: Nice that you use an existing plugin.

File: Graphe_chartjs/index.js [hunk 2]
```diff
@@ -1,56 +1,71 @@
...
+      Chart.register(autocolors);  //Enregistrement de tous les graphes
+      const lighten = (color, value) => Chart.helpers.color(color).lighten(value).rgbString();
+       
+      // données creation graphique mode transport pour chaque univ
+      console.log(JSONdata.data)
```
Comment (damienmarchal) [inline Graphe_chartjs/index.js]: Remove console.log/debugging facilities before merging

File: Graphe_chartjs/index.js [hunk 3]
```diff
@@ -1,56 +1,71 @@
...
-      function(index){
-        return Object.values(index)[0];
-      }) 
-  
-      console.log(labels)
-      console.log(data_global)
+      const donnee  = JSONdata.data.map( index => {
+          return Object.values(index)[0];  //parcours une liste d'ensemble(dictionnaire) et recupère les valeurs
+        })
+      console.log(donnee)
```
Comment (damienmarchal) [inline Graphe_chartjs/index.js]: Same here.

File: Graphe_chartjs/index.js [hunk 4]
```diff
@@ -1,56 +1,71 @@
...
-  
-      console.log(labels)
-      console.log(data_global)
+      const donnee  = JSONdata.data.map( index => {
+          return Object.values(index)[0];  //parcours une liste d'ensemble(dictionnaire) et recupère les valeurs
+        })
+      console.log(donnee) 
+      
+      const labels = Object.keys(donnee[0]) ////recupère clés d'un ensemble(dictionnaire) sous forme de liste
+      console.log(labels)
```
Comment (damienmarchal) [inline Graphe_chartjs/index.js]: And here.... and few other places

**Comment by damienmarchal** [comment]
Hello @cosnelle,

Can you please add a description with a picture to this PR. 
This PR is including serious algorithm, clean implementation and the final rendering of the graph look very good. 
There is several minor changes to do related to printing/debugging that I suggest to remove before merging. 

@juridolfi  can you makes a review on the general code  if there is things that catch your eyes ?

**Comment by juridolfi** [comment]
@cosnelle J'ai relu le code, tout est bon pour moi

**Pull request merged**